server/service/src/sync/program_schemas/hiv_care_program.json

```diff
@@ -96,28 +96,38 @@
     },
     "HIVCareProgramEnrolment": {
       "properties": {
+        "clinicTransferredFrom": {
+          "type": "string"
+        },
+        "dateTransferredIn": {
+          "format": "date",
+          "type": "string"
+        },
         "enrolmentDatetime": {
           "description": "Enrolment date and time",
           "format": "date-time",
           "type": "string"
         },
-        "enrolmentPatientId": {
-          "description": "Program enrolment specific patient id.",
-          "type": "string"
-        },
         "hivConfirmationDate": {
           "format": "date",
           "type": "string"
         },
         "hivTestType": {
           "$ref": "#/definitions/HIVTestType"
         },
-        "mother": {
-          "$ref": "#/definitions/HIVRelation"
```

Comment:
I think we should keep the more general case of having the whole person object. In the UI schema you just record the mother id though. In this way we stay much more flexible. For example, if we need more person information in the future we already have a place to store it and we just need to change the UI.

Reply:
both changes here (: thanks Clemens!
[1cf985e](https://github.com/openmsupply/open-msupply/pull/1215/commits/1cf985ed6388b04469981f45732477a20cfab228)